ex_feels/lib/ex_feels_web/templates/layout/app.html.eex

```diff
@@ -9,6 +9,86 @@
 
     <title>BitFeels</title>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
+    <style type="text/css">
+      @import url(//fonts.googleapis.com/css?family=Open+Sans:400,700);
```

Comment:
Lets remove this and the `svg` style. Fonts should be global and handled elsewhere imo.

Reply:
done